common/src/main/java/com/wynntils/models/items/encoding/impl/block/CustomConsumableTypeDataTransformer.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.items.encoding.impl.block;
+
+import com.wynntils.models.gear.type.ConsumableType;
+import com.wynntils.models.items.encoding.data.CustomConsumableTypeData;
+import com.wynntils.models.items.encoding.type.DataTransformer;
+import com.wynntils.models.items.encoding.type.ItemTransformingVersion;
+import com.wynntils.utils.type.ArrayReader;
+import com.wynntils.utils.type.ErrorOr;
+import com.wynntils.utils.type.Pair;
+import com.wynntils.utils.type.UnsignedByte;
+import java.util.List;
+
+public class CustomConsumableTypeDataTransformer extends DataTransformer<CustomConsumableTypeData> {
+    private static final List<Pair<ConsumableType, Integer>> CONSUMABLE_TYPE_IDS = List.of(
+            new Pair<>(ConsumableType.POTION, 0),
```

Comment:
Here's another one of the type/enum situations. Marking for reference; I'll get back to the general discussion on how to handle these separately.

Reply:
You are reviewing an older commit, this is fixed now.